modules/calib3d/src/usac/utils.cpp

```diff
@@ -169,6 +169,9 @@ class SolvePoly : public SolverPoly {
     int getRealRoots (const std::vector<double> &coeffs, std::vector<double> &real_roots) override {
         if (coeffs.empty())
             return 0;
+        for (auto c : coeffs)
+            if (std::isnan(c))
```

Comment:
`cvIsNaN` (which properly works in case of `-ffast-math`)

---

Inf?

Reply:
fixed